packages/libs/src/sdk/api/generator/gen.js

```diff
@@ -39,23 +61,63 @@ const generate = ({ env, apiVersion, apiFlavor, generator }) => {
     // Hardcode a prod DN, it doesn't matter
     baseURL = 'https://discoveryprovider.audius.co'
   }
-  const outputFolderName = apiFlavor === '' ? 'default' : apiFlavor
   const apiPath = apiFlavor === '' ? apiVersion : `${apiVersion}/${apiFlavor}`
 
+  const res = await fetch(`${baseURL}/${apiPath}/swagger.json`)
+  const json = await res.text()
+  fs.writeFileSync(path.join(process.env.PWD, SWAGGER_JSON_PATH), json)
+}
+
+const upgradeSpec = async () => {
+  const openApiGeneratorArgs = [
+    'generate',
+    '-g',
+    'openapi',
+    '-i',
+    `/local/${SWAGGER_JSON_PATH}`,
+    '-o',
+    `/local/${OUT_DIR}`,
+    '--skip-validate-spec'
+  ]
+  await spawnOpenAPIGenerator(openApiGeneratorArgs)
+}
+
+const processSpec = () => {
+  const swagger = JSON.parse(
+    fs.readFileSync(path.join(process.env.PWD, SWAGGER_JSON_PATH), 'utf-8')
+  )
+  const openApi = JSON.parse(
+    fs.readFileSync(path.join(process.env.PWD, OPEN_API_JSON_PATH), 'utf-8')
+  )
+
+  for (const [key, value] of Object.entries(swagger.definitions)) {
```

Comment:
insane

Reply:
Oh yeah ouch 😬 
So if we wanted to add more 3.0 features, we'd need to insert them here as well?